balena/models/tag.py

```diff
@@ -339,23 +341,25 @@ def get_all_by_application(self, app_id):
 
         return super(ReleaseTag, self).get_all(raw_query=query)
 
-    def get_all_by_release(self, release_id):
+    def get_all_by_release(self, commit_or_id):
         """
         Get all release tags for a release.
 
         Args:
-            release_id (str): release id.
+            commit_or_id: release commit (str) or id (int).
 
         Returns:
             list: list contains release tags.
 
         Examples:
-            >>> balena.models.tag.release.get_all_by_release('135')
+            >>> balena.models.tag.release.get_all_by_release(135)
```

Comment:
Based on the example result, this should be
```suggestion
            >>> balena.models.tag.release.get_all_by_release(465307)
```
Could we also have an example using the commit hash as an argument?